easybuild/easyconfigs/g/gnuplot/gnuplot-5.2.5-foss-2018b.eb

```diff
@@ -35,6 +35,7 @@ dependencies = [
     ('Pango', '1.42.4'),
     ('libcerf', '1.7'),
     ('Qt5', '5.10.1'),
+    ('Lua', '5.2.4'),
```

Comment:
@branfosj Seems like we should stick to Lua 5.1.5 here to avoid introducing two `Lua` dep variants?

`ARGoS-3.0.0-beta53-foss-2018b-Lua-5.2.4.eb` is already an exception since the newer Lua dep is indicated in the `versionsuffix`...